flambe/nlp/classification/__init__.py

```diff
@@ -1,5 +1,6 @@
 from flambe.nlp.classification.datasets import SSTDataset, TRECDataset, NewsGroupDataset
-from flambe.nlp.classification.model import TextClassifier
+from flambe.nlp.classification.task import TextClassification, TextClassifier
```

Comment:
This is a little bit confusing... is `TextClassifier` a task?

Reply:
I can put them in separate modules. TextClassification is the task.